Library/Formula/srclib.rb

```diff
@@ -0,0 +1,192 @@
+  end
+
+  go_resource "github.com/sourcegraph/go-github" do
+    url "https://github.com/sourcegraph/go-github",
+        :revision => "c6edc3e74760ee3c825f46ad9d4eb3e40469cb92", :using => :git
+  end
+
+  go_resource "github.com/sourcegraph/httpcache" do
+    url "https://github.com/sourcegraph/httpcache",
+        :revision => "e2fdd7ddabf459df5cd87bd18a4616ae7084763e", :using => :git
+  end
+
+  go_resource "github.com/sourcegraph/mux" do
+    url "https://github.com/sourcegraph/mux",
+        :revision => "dd22f369d469f65c3946889f5d8a1fb3933192e9", :using => :git
+  end
+
+  go_resource "github.com/sqs/fileset" do
+    url "https://github.com/sqs/fileset",
+        :revision => "4317e899aa9438ba7603a6e322389571cb3ffdff", :using => :git
+  end
+
+  go_resource "golang.org/x/tools" do
+    url "https://go.googlesource.com/tools",
+        :revision => "a18bb1d557dac8d19062dd0240b44ab09cfa14fd", :using => :git
+  end
+
+  go_resource "sourcegraph.com/sourcegraph/go-diff" do
+    url "https://github.com/sourcegraph/go-diff",
+        :revision => "07d9929e8741ec84aa708aba12a4b1efd3a7a0dd", :using => :git
+  end
+
+  go_resource "sourcegraph.com/sourcegraph/go-flags" do
+    url "https://github.com/sourcegraph/go-flags",
+        :revision => "f819544216a8b66157184f0976948f92a8144fe7", :using => :git
+  end
+
+  go_resource "sourcegraph.com/sourcegraph/go-nnz" do
+    url "https://github.com/sourcegraph/go-nnz",
+        :revision => "62f271ba06026cf310d94721425eda2ec72f894c", :using => :git
+  end
+
+  go_resource "sourcegraph.com/sourcegraph/go-sourcegraph" do
+    url "https://github.com/sourcegraph/go-sourcegraph",
+        :revision => "6f1cd4d2b721cff7913ed2f04bcd820590ce3b94", :using => :git
+  end
+
+  go_resource "sourcegraph.com/sourcegraph/go-vcs" do
+    url "https://github.com/sourcegraph/go-vcs",
+        :revision => "1dcc4655df7318c3f105f9212900e1d0c68f7424", :using => :git
+  end
+
+  go_resource "sourcegraph.com/sourcegraph/makex" do
+    url "https://github.com/sourcegraph/makex",
+        :revision => "ba5e243479d710a5d378c97d007568a405d04492", :using => :git
+  end
+
+  go_resource "sourcegraph.com/sourcegraph/rwvfs" do
+    url "https://github.com/sourcegraph/rwvfs",
+        :revision => "451122bc19b9f1cdfeb2f1fdccadbc33ef5aa9f7", :using => :git
+  end
+
+  go_resource "sourcegraph.com/sourcegraph/vcsstore" do
+    url "https://github.com/sourcegraph/vcsstore",
+        :revision => "53d0c58fd11f7dc451456eb983050c58cd005268", :using => :git
+  end
+
+  def install
+    ENV["GOBIN"] = bin
+    ENV["GOPATH"] = buildpath
+
+    mkdir_p buildpath/"src/sourcegraph.com/sourcegraph"
+    ln_sf buildpath, buildpath/"src/sourcegraph.com/sourcegraph/srclib"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd "cmd/src" do
+      system "go", "build", "-o", "src"
+      bin.install "src"
+    end
+  end
+
+  test do
+    system "git", "clone", "https://github.com/sourcegraph/srclib-sample"
```

Comment:
Make this a regular `resource` and stage it in the test block rather than directly doing `git clone`, please!